hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/MapInfoCollector.java

```diff
@@ -65,4 +66,15 @@ private long countMapWithMapStoreEnabled(Node node) {
             return false;
         }).count();
     }
+
+    private long countMapWithAtleastOneQueryCache(Node node) {
+        return maps.stream().filter(distributedObject -> {
+            MapConfig config = node.getConfig().getMapConfig(distributedObject.getName());
+            if (config != null) {
+                return config.getQueryCacheConfigs().size() >= 1;
```

Comment:
Please write it as `!config.getQueryCacheConfigs().isEmpty()` instead of checking the size.